samples/basic/helloworld/test.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+test_mode=$1
+run_mode=$2
+
+set -e
+
+if [[ "$test_mode" == "clean" ]]; then
+    make clean
+elif [[ "$test_mode" == "init" ]]; then
+    make
+elif [[ "$test_mode" == "run" ]]; then
+    make "$run_mode"
+elif [[ "$test_mode" == "gettimeout" ]]; then
+    # Default
+    exit 1
```

Comment:
This is 1 second and it is too short to be default value. May be 60 seconds is better.

Reply:
No, this means "default". This is how it works in the case of the Makefiles of the tests. If there is no target called ``gettimeout``, ``make`` will return 1. If the target is there, it prints the timeout length.

Take a look at ``samples/ltp/batch.mk``, target ``gettimeout``, for example/